cms/envs/common.py

```diff
@@ -2666,3 +2666,5 @@
 
 # keys for  big blue button live provider
 COURSE_LIVE_GLOBAL_CREDENTIALS = {}
+
+PERSONALIZED_RECOMMENDATION_COOKIE_NAME = 'edx-user-personalized-recommendation'
```

Comment:
@shafqatfarhan Small nit, take it or leave: pluralize the word recommendation in both variable and its value. e.g. `recommendations`. If we do so, we might need to change it inside other repos as well where we are using it to get the cookie.

Reply:
I kept this singular because this cookie name variable refers to a singular object and in convention, we only use plural variable names if it represents a collection i.e list/dictionary.